src/components/raycaster.js

```diff
@@ -66,11 +66,26 @@ module.exports.Component = registerComponent('raycaster', {
     var intersectedObjs = this.intersect(scene.children);
     for (var i = 0; i < intersectedObjs.length; ++i) {
       intersectedObj = intersectedObjs[i];
+
+      while (
+        (intersectedObj.object.parent) &&
```

Comment:
we can put this if statement on one line without the parens

and comment the while loop ```//  Iterate up to parent when matching subobject.```